src/BikeSharing.DomainLogic.Tests/TrainerTests.cs

```diff
@@ -8,138 +9,234 @@ namespace Trainer.Tests
 {
 
     [TestClass]
-    public class WorkoutTests
+    public class TestAthlete
     {
-        User _user;
+        Athlete athlete;
```

Comment:
keeping around state in test classes in not recommended.  Just move this to be a local vairable in your test methods.